manifests/init.pp

```diff
@@ -737,6 +745,14 @@
     }
   }
 
+  if $::selinux and $manage_selinux_real and $sshd_config_port != '22' {
```

Comment:
SELinux comes with that label by default so just avoiding the exec on the machine if it's still on port 22.  I could see an argument to just always include it too though - either way.